ecommerce/android/Getflix/app/src/main/java/com/example/getflix/ui/adapters/OrderProductsAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.getflix.ui.adapters
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.example.getflix.databinding.OrderProductItemBinding
+import com.example.getflix.models.OrderPurchasedModel
+import com.example.getflix.ui.fragments.BankAccountFragment
+import com.example.getflix.ui.fragments.OrderInfoFragment
+import com.example.getflix.ui.fragments.OrderProductsFragment
+import com.example.getflix.ui.viewmodels.OrderPurchasedViewModel
+import com.example.getflix.ui.fragments.OrderInfoFragmentDirections
+
+class OrderProductsAdapter(
+    private val orderPurchasedList: ArrayList<OrderPurchasedModel>?
+) : ListAdapter<OrderPurchasedModel, OrderProductsAdapter.RowHolder>(OrderProductsDiffCallback()) {
+
+
+
+    class RowHolder(val binding: OrderProductItemBinding) : RecyclerView.ViewHolder(binding.root) {
+
+        fun bind(orderPurchased: OrderPurchasedModel, position: Int) {
+            binding.listproduct = orderPurchased
+        }
```

Comment:
After you initialize the data in the layout, you don't the bind the price of the order neither in the bind function of RowHolder nor in the `order_product_item.xml`

Reply:
now i am pushing the revised version which binded